### PR TITLE
Fixes App Icon download and copy for Plugins. 

### LINKED ISSUE #4241: Broken links in the docs
## Summary
There are two broken links in the documentation which I don't know how to fix.

### Audience
Teleport users

### Location

- Mattermost Integration
- Slack Integration

Search for "Download Teleport Bot Icon" on both pages. It's unclear what is this supposed to be? It doesn't really fit into the context around it either.

Another problem (on both pages) is the following paragraph:

_"We'll reference these later when configuring Teleport-Plugins."_ (it's unclear where is it linking?)

### PR DESCRIPTION
**Pull request by benarent**
This Fixes #4241

I've fixed the broken links for the plugin icons and also updated the language. It could still be improved further but it's getting there. Also fixed a milv error.

### CODE REVIEW

File: docs/4.3/enterprise/workflow/ssh_approval_slack.md [hunk 1]
```diff
@@ -191,7 +190,7 @@ You can create a test permissions request with `tctl` and check if the plugin wo
 #### Create a test permissions request behalf of a user.
 
 ```bash
-# Replace USERNAME with a local user, and TARGET_ROLE with a Teleport Role
+# Replace USERNAME with a Teleport local ser, and TARGET_ROLE with a Teleport Role
```
Comment (webvictim) [inline docs/4.3/enterprise/workflow/ssh_approval_slack.md:193]:
```suggestion
# Replace USERNAME with a Teleport local user, and TARGET_ROLE with a Teleport Role
```

File: docs/4.3/milv.config.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ request-repeats: 3
 timeout: 5
 allow-redirect: true
 allow-code-blocks: false
-ignore-external: false
+ignore-external: true
```
Comment (webvictim) [inline docs/4.3/milv.config.yaml:5]: Commented about this on the other PR.

**Comment by benarent** [comment]
Something strange is happening, `milv` is working Ok locally

```
➜  4.3 git:(docs/benarent/slackbot-img) milv
NO ISSUES :-)
```

**Comment by webvictim** [comment]
Yeah, I think the reason is because Drone takes your PR and merges it with the current state of `master` before running all the tests (to make sure that it merges cleanly). We committed a broken `milv` state to `master`, which meant that every single PR was failing because of that change. I've merged the fix PR now which means all the Drone tests will pass again.

File: docs/4.3/architecture/authentication.md [hunk 3]
```diff
@@ -226,9 +226,9 @@ back-ends as shown in the table below:
 
 Data Type        | Supported Back-ends       | Notes
 -----------------|---------------------------|---------
-Cluster state    | `dir`, `etcd`, `dynamodb` | Multi-server (HA) configuration is only supported using `etcd` and `dynamodb` back-ends.
-Audit Log Events | `dir`, `dynamodb`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
-Recorded Sessions| `dir`, `s3`               | `s3` is mandatory if `dynamodb` is used for the audit log.
+Cluster state    | `dir`, `etcd`, `dynamodb`,`firestore` | Multi-server (HA) configuration is only supported using `etcd`, `dynamodb`, and `firestore` back-ends.
+Audit Log Events | `dir`, `dynamodb`, `firestore`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
```
Comment (webvictim) [inline docs/4.3/architecture/authentication.md:230]: I think I asked this on the other PR, but do we have a similar requirement that you **must** use Google storage for recorded sessions if you use `firestore` for audit logs? I have to confess that I don't completely understand why we have this hard requirement for `dynamodb` either.

**Review by webvictim — APPROVED** [review]

File: docs/4.3/architecture/authentication.md (hunk 3 above)
Comment (benarent) [reply docs/4.3/architecture/authentication.md:230]: That's a good question. I don't think so.. when I was trying it out a while ago but I'll make a note to double check. I think the reason for using dynamodb/s3 is for HA and keeping it consistent but I can see a reason to still keep them on disk.

**Comment by benarent** [comment]
Thanks for the drone / milv clarification gus

**Pull request merged**